packages/odata-service-inquirer/src/translations/odata-service-inquirer.i18n.json

```
@@ -55,17 +55,17 @@
         "validationMessages": {
             "odataVersionMismatch": "The template you have chosen supports V{{requiredOdataVersion}} OData services only. The provided version is V{{providedOdataVersion}}.",
             "metadataInvalid": "The service metadata is invalid.",
-            "metadataFilePathNotValid": "Metadata file does not exist or is not accessible. Please specify a valid file path.",
-            "capProjectNotFound": "The folder you have selected does not seem to contain a valid CAP project. Please check and try again.",
-            "warningCertificateValidationDisabled": "User has disabled certificate validation",
-            "annotationsNotFound": "Annotations not found for specified service"
+            "metadataFilePathNotValid": "The metadata file does not exist or is not accessible. Please specify a valid file path.",
+            "capProjectNotFound": "The folder you have selected does not contain a valid CAP project. Please check and try again.",
+            "warningCertificateValidationDisabled": "You have disabled certificate validation. You must ensure the certificate is valid.",
+            "annotationsNotFound": "Annotations not found for the specified service."
         },
         "warnings": {
-            "nonUIServiceTypeWarningMessage": "Please note that {{serviceType}} services, or not classified services, are not intended to be used for the generation of SAP Fiori UI applications",
-            "noServicesAvailable": "No services available for the selected system, see logs for further details.",
-            "noServicesAvailableForOdataVersion": "There are no V{{odataVersion}} OData services available from the selected system and the template you have chosen supports V{{odataVersion}} OData services only",
-            "noAnnotations": "No backend annotations associated with this service were retrieved and may result in an invalid application being created",
-            "collaborativeDraftMessage": "The selected service is draft enabled but does not support collaborative draft."
+            "nonUIServiceTypeWarningMessage": "{{serviceType}} services and non-classified services are not intended to be used for generating SAP Fiori UI applications. Your application may not work as expected.",
```

`NOT_CLASSIFIED` is a technical term associated to some services so we should maintain the term as such.
```suggestion
            "nonUIServiceTypeWarningMessage": "Services of service type: {{serviceType}} or without a service type assignment are not intended to be used for generating SAP Fiori UI applications. Your application may not work as expected.",
```